v2/internal/controllers/crd_machinelearningservices_test.go

```diff
@@ -27,16 +27,12 @@ func Test_MachineLearning_Workspaces_CRUD(t *testing.T) {
 	rg := tc.CreateTestResourceGroupAndWait()
 
 	sa := newStorageAccount(tc, rg)
-
-	tc.CreateResourceAndWait(sa)
-
-	kv := newVault(tc, rg)
-	tc.CreateResourceAndWait(kv)
+	kv := newVault("kv", tc, rg)
```

Comment:
curious: Why did we need the name change here? Was it because there was an old KeyVault somewhere that we had a hard time finding to purge?

Reply:
Yeah - after spending 20+m trying to find it, I figured a name change was easier.

Reply:
I did it this way to avoid changing the other recordings.